conan/tools/files/files.py

```diff
@@ -66,18 +66,19 @@ def rmdir(conanfile, path):
     _internal_rmdir(path)
 
 
-def rm(conanfile, pattern, folder, recursive=False):
+def rm(conanfile, pattern, folder, recursive=False, excludes=None):
     """
     Utility functions to remove files matching a ``pattern`` in a ``folder``.
 
     :param conanfile: The current recipe object. Always use ``self``.
     :param pattern: Pattern that the files to be removed have to match (fnmatch).
     :param folder: Folder to search/remove the files.
     :param recursive: If ``recursive`` is specified it will search in the subfolders.
+    :param excludes: Pattern to not be removed when matched with the pattern.
```

Comment:
In ``copy()`` this is:

>  :param excludes: (Optional, defaulted to ``None``) A tuple/list of fnmatch patterns or even a
           single one to be excluded from the copy.

I think it would be good to use the same logic, and use a list/tuple here for exclusions, not just 1 single value.